pino.js

```diff
@@ -382,6 +361,13 @@ function pino (opts, stream) {
 
 tools.defineLevelsProperty(pino)
 
+function extreme (dest) {
+  if (!dest) {
+    return new SonicBoom(process.stdout.fd)
```

Comment:
didn't we support a thing that allows for `process.stdout` and `process.stderr` to be the dest? 
Does `SonicBoom` support that, or only fd's?